sentry/templates/configmap-nginx.yaml

```diff
@@ -42,6 +42,7 @@ data:
       }
 
       location /_static/ {
+        proxy_pass http://sentry;
```

Comment:
I think that would only work if the chart name (and therefore the service name) is "sentry"

Reply:
This will work. This is the name of the upstream, not the service